src/binding/transaction_log_file.h

```diff
@@ -33,13 +35,15 @@
 #include <sys/stat.h>
 
 #define TRANSACTION_LOG_TOKEN 0x574f4f46
+#define POSITION_OF_LOG_FILE_TIMESTAMP 5
```

Comment:
I've been trying to name all the preprocessor macros with a prefix of the context, so something like this:

```suggestion
#define TRANSACTION_LOG_FILE_TIMESTAMP_POSITION 5
```